OpenEdXMobile/src/main/java/org/edx/mobile/util/images/ShareUtils.java

```diff
@@ -127,15 +129,49 @@ public boolean onMenuItemClick(MenuItem item) {
         menuHelper.show();
     }
 
+    public static void showCelebrationShareMenu(Activity activity, View anchor,
+                                                EnrolledCoursesResponse course,
+                                                ShareTypeSelectedListener shareTypeSelectedListener) {
+        final String shareTextWithPlatformName = ResourceUtil.getFormattedString(
+                activity.getResources(),
+                R.string.celebration_share_message, new HashMap<String, CharSequence>() {{
+                    put("course_name", course.getCourse().getName());
+                    put("platform_name", activity.getString(R.string.platform_name));
+                }}).toString() + "\n\n" + course.getCourse().getCourse_about();
+        ShareUtils.showShareMenu(activity, ShareUtils.newShareIntent(shareTextWithPlatformName), anchor, (componentName, shareType) -> {
+            final String shareText;
```

Comment:
can we define it like this
`final String shareText = shareTextWithPlatformName + 
                    (shareType != ShareType.UNKNOWN ?  
                            "?utm_campaign=" + activity.getString(R.string.platform_name).toLowerCase() +
                                    "milestone&utm_medium=social&utm_source=" + shareType.utmParamKey : "");`

Reply:
??

Reply:
updated.